chromium_src/chrome/installer/mini_installer/mini_installer.cc

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2020 The Brave Authors
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#if defined(BRAVE_CHROMIUM_BUILD) && BUILDFLAG(ENABLE_BRAVE_REFERRALS)
```

Comment:
the `BRAVE_CHROMIUM_BUILD` check is not needed here, only `ENABLE_BRAVE_REFERRALS`

Reply:
also you need the header for the buildflag here `#include "brave/components/brave_referrals/buildflags/buildflags.h"`